frontend/projects/ui/src/app/pages/server-routes/server-show/server-show.page.html

```diff
@@ -15,15 +15,20 @@
 
   <!-- loaded -->
   <ion-item-group *ngIf="server$ | async as server; else loading">
-    <ion-item *ngIf="!secure" color="warning">
+    <ion-item *ngIf="!secure || isTorHttp" color="warning">
       <ion-icon slot="start" name="warning-outline"></ion-icon>
       <ion-label>
-        <h2 style="font-weight: bold">You are using unencrypted http</h2>
+        <h2 style="font-weight: bold">Http detected</h2>
         <p style="font-weight: 600">
-          Click the button on the right to switch to https. Your browser may
-          warn you that the page is insecure. You can safely bypass this
-          warning. It will go away after you download and trust your server's
-          certificate
+          {{ isTorHttp ? 'Tor is faster over https.' : 'Your connection is
```

Comment:
What's the reasoning behind the claim that "Tor is faster over HTTPs"? All it does is adding another round-trip to establish a secure connection over an already secure connection. 

I'd get it if it was about accessing the clearnet version via HTTPs over Tor, that might reduce the latency (no Tor between rendezvous point and hidden service), but as it is it's just accessing the hidden service with an extra layer of TLS.

Reply:
It's faster because browsers only support http2 over https. http2 allows better batching and tcp connection reuse than 1.1 which results in faster page load times

Reply:
TIL, thx!